### DISCUSSION TITLE
友情链接

### DISCUSSION THREAD

**Discussion by shenweiyan** [shenweiyan/Digital-Garden#5] [0.1-站点]
> 每一个独立博客都是一个信息孤岛。我们没有类似「你可能喜欢其他人写的这些文章」的推荐机制，所以除了搜索引擎、社交网络引荐之外，我们应该还需要通过某种方法将这些信息孤岛连接起来：**交换友情链接就是一种很棒的架桥方式。**

社交就是不断认识新的人、结交新的朋友，而友链也正是这样一种很棒的交朋友的方式，作为一个不太善于言辞的技术人，我希望结交更多的朋友，来丰富我的社交，一起成长。

## 站点信息

**申请前记得先添加本站哦**，站点信息如下。

- 站点名称: 维燕的数字花园
- 站点链接: <https://shenwy.com>
- 站点头像: <https://shenwy.com/assets/logo.png>
- 站点描述: BioIT 爱好者，沉淀生活，记录点滴。

## 交换友链

想和我交换友链的，没啥特别需求，只是希望你的博客有足够多的**原创**、**有意义**的内容，并且建站已满半年。

- 不接受商业性或包含侵入性广告的站点。
- 那种塞满了转载文章的采集站点暂时不考虑。
- 非常反感很多论坛上动不动就来交换友链的帖子，如果是出于 SEO 的一些目的来交换友链其实是没什么意义的。

## 友链说明

如需申请，在下方评论区提供你的「博客名称」和「博客地址」即可。或者参考以下格式：

- 博客名称：xxxxx
- 博客地址：https://xxxx.com
- 博客介绍：xxxxxxxx

## 友情站点

以下站点按照添加的时间顺序排列。

- [**JiaoYuan's blog**](https://jiaoyuan.org/)
  https://avatars.githubusercontent.com/u/105262243
  思绪来得快去得也快，偶尔会在这里停留。

- [**1874’s Blog**](https://blog.1874.cool)
  https://avatars.githubusercontent.com/u/37357188
  有粤语歌就不会有世界末日。

- [**Jayden's site**](https://xxu.do/)
  https://avatars.githubusercontent.com/u/97662597
  为天地立心，为生民立命，为往圣继绝学，为万世开太平。

- [**Shitao Wu | 吴诗涛**](https://shitao5.org/)
  https://shitao5.org/logo.png
  喜欢捣鼓电脑，UseR，喜欢[阅读](https://shitao5.org/tags/%E8%AF%BB%E5%90%8E%E6%84%9F/)，重度使用 Kindle。甘愿做一个平庸的人，走在成为我自己的道路上。

- [**椒盐豆豉**](https://blog.douchi.space/)
  https://douchi.sfo3.cdn.digitaloceanspaces.com/blog/dino_statict.png
  一个写了二十年博客，现居美国西雅图的女博主+码农。

- [**Pepper 的博客**](https://uioqps.github.io/)
  https://uioqps.github.io/me.jpg
  生命留给有价值的事物才能创造价值。

- [**Wanjin Hu (胡万金)**](https://wanjinhu.github.io/)
  https://avatars.githubusercontent.com/u/69741445
  思考人生，探索世界，知行合一。

- [**Wcowin's Blog**](https://wcowin.work/)
  https://pica.zhimg.com/80/v2-74ecd899c7c4cc0258930eaff239a21b_1440w.webp
  循此苦旅，以达星辰。

- [**Goslion 的博客**](https://goslion.eu.org/)
  https://avatars.githubusercontent.com/u/117809044
  一个专注技术与写作的个人博客，记录日常与技术。

- [**博友圈**](https://www.boyouquan.com/home)
  https://www.boyouquan.com/assets/images/sites/logo/logo-small.svg
  博客人的朋友圈，将一个个散落在各处的孤岛连接成一片广袤无垠的新大陆！

- [**极简主义**](https://hongjun.vip/)         
  https://hongjun.vip/assets/avatar.png
  文档即产品。

**Comment by shenweiyan** [comment]
#### 失效站点信息



- [**望薮**](https://ralvines.top)
  以我观物，故物皆著我之色彩。

- [**墨迹心空**](https://www.secondlife.love/)
  https://avatars.githubusercontent.com/u/116415567
  Life is coding.

**Comment by Jaaayden** [comment]
博客名称：Jayden
博客地址：https://xxu.do
博客介绍为天地立心,为生民立命, 为往圣继绝学,为万世开太平。

**Reply by shenweiyan** [reply]
已经加上了 >.<

**Comment by vbskycn** [comment]
hello

**Comment by secondlife1127** [comment]
站名：墨迹心空
站点链接：https://www.secondlife.love
站点介绍：life is coding

**Reply by shenweiyan** [reply]
已加 >.<

**Comment by uioQPS** [comment]
你好，uioqps.github.io
可以交流下技术相关的问题吗

**Reply by shenweiyan** [reply]
友链已加，欢迎多多指教💖

**Comment by wanjinhu** [comment]
博客名称：Wanjin Hu (胡万金)
博客地址：https://wanjinhu.github.io
博客介绍：思考人生，探索世界，知行合一。

我的主页比较单调，平时更新的并不是很多，一般是和我专业相关的长文

**Reply by shenweiyan** [reply]
已加，胡老师有空也麻烦给我的站点加个友链啦 >.<

**Reply by wanjinhu** [reply]
Hi, Weiyan 

友情链接已加。

**Comment by DawnEve** [comment]
博客名称：BioMooc
博客地址：https://www.biomooc.com/
博客介绍：R语言可视化为主的生信网站

**Reply by shenweiyan** [reply]
看了一下，您的站点广告付费的服务有点多哦，分享的知识内容也很多 >.<

**Reply by DawnEve** [reply]
没有付费内容，只是引用了谷歌广告，目前连域名费都覆盖不了

**Comment by Wcowin** [comment]
>名称: Wcowin's Blog  
>链接: https://wcowin.work/  
>头像: https://s2.loli.net/2024/02/01/gaE47y5fKM6kosV.png  
 >简介: 循此苦旅，以达星辰

已添加！学习了很多~RSS那篇文章解决了我的疑惑

**Reply by shenweiyan** [reply]
已加，站点更新后就能在页面看到了。

**Comment by ssttyyxxx** [comment]
名称：Goslion的博客
链接：https://goslion.eu.org/
头像：无
简介：一个专注技术与写作的个人博客，记录日常与技术

**Reply by shenweiyan** [reply]
已加，站点更新后就可以看到了🤝

**Comment by lzh1230** [comment]
网站名称：拾趣博客导航
网站链接：https://s7.fan/
网站图标：https://s7.fan/img/tubiao.png
网站简介：捡拾文字里的小乐趣！

**Reply by shenweiyan** [reply]
很抱歉，友链暂时不考虑导航站点。
